src/libs/ReportUtils.ts

```diff
@@ -5274,8 +5274,6 @@ function shouldReportBeInOptionList({
         report?.reportName === undefined ||
         // eslint-disable-next-line @typescript-eslint/prefer-nullish-coalescing
         report?.isHidden ||
-        // eslint-disable-next-line @typescript-eslint/prefer-nullish-coalescing
-        participantAccountIDs.includes(CONST.ACCOUNT_ID.NOTIFICATIONS) ||
```

Comment:
@mountiny [Replying](https://github.com/Expensify/App/pull/41343#issuecomment-2157687834) - I've just fixed it. But I need to confirm - is there any other situation that we want to hide this kind of chat (including persona as a participant) from LHN? If there is any I need to think of changing this confition

Reply:
I think it was added here https://github.com/Expensify/App/pull/32361 but now we can show it at least in the LHN. You should still not bae able to start a new chat with the notification@expensify.com account

Reply:
Even when a user accountID is even, we show Expensify persona. 

https://github.com/Expensify/App/assets/25876548/5caf77cd-7608-42d6-b498-a3d0811cf9f5

Do we want to show chat with Expensify persona for the first time only? Do we want to always show it (even after user logs in to already active account)?

Reply:
@mountiny yeah, but even before my latest changes we had this account displayed when starting a new chat
<img width="373" alt="Screenshot 2024-06-10 at 11 26 22" src="https://github.com/Expensify/App/assets/28020445/002d2552-9582-4ed7-aaa7-491df730dff8">

> Do we want to show chat with Expensify persona for the first time only? Do we want to always show it (even after user logs in to already active account)?
- and I also wonder what is expected behavior here? Should we show the Expensify persona chat for all the users?

Reply:
I think the logic is that in the onboarding flow, some will get the tasks from Concierge and some from the Expensify persona.

It should only show the Expensify persona for those who have the onboarding flow added to the persona chat. Not for the others. I now realize this might be tricky cc @francoisl 

> yeah, but even before my latest changes we had this account displayed when starting a new chat

I think that is a bug and it should not show up in any of those lists

Reply:
@sobitneupane are you sure - I've just created 2 accounts and for one I have only the chat with persona shown for the other one the Concierge chat is only shown 
<img width="1511" alt="Screenshot 2024-06-10 at 16 12 56" src="https://github.com/Expensify/App/assets/28020445/8fc8b5b0-ad7e-46cc-929b-5021fe99c86d">

has it been fixed on the BE? cc: @mountiny

Reply:
No changes in backend as far as I know

Reply:
<img width="370" alt="Screenshot 2024-06-10 at 17 10 04" src="https://github.com/Expensify/App/assets/28020445/e86f59f2-14fe-4c0c-b8eb-2e770833f8a0">

notifications@expensify.com is visible even if I revert the latest changes, so I need to add a new filter.
For displaying the Expensify persona only for the users onboarded by persona - I still need to figure this out.

Reply:
@mountiny should this persona chat be always visible, or it should disappear at some point?

Reply:
> is there any other situation that we want to hide this kind of chat

Users should not be able to create group chats with the system account, or invite it to existing rooms, workspaces, etc. For example, the "Add to group" button shouldn't appear here for the Expensify row (maybe we can look into that in a follow-up PR if that requires a lot of changes)

<img width="389" alt="image" src="https://github.com/Expensify/App/assets/2229301/3fca1803-0e54-4a6c-8586-e2bae7fac80e">

----


> It should only show the Expensify persona for those who have the onboarding flow added to the persona chat. Not for the others. I now realize this might be tricky cc @francoisl

Yeah I don't think it's something we had considered originally. To be clear though, do you mean that **EITHER** Concierge or the system chat would remain sticky in the LHN when you're in "Most Recent" mode? Concierge is pinned by default (I unpinned it here to show that it remains in the LHN), but yes I think I agree that if the onboarding tasks are created in the Concierge chat, then it just adds extra noise in the LHN to also show the system DM, so it would make sense to hide it.

<img width="371" alt="image" src="https://github.com/Expensify/App/assets/2229301/da973487-8687-41b9-b41f-b974a35aa0f5">

Reply:
> Yeah I don't think it's something we had considered originally. To be clear though, do you mean that EITHER Concierge or the system chat would remain sticky in the LHN when you're in "Most Recent" mode? Concierge is pinned by default (I unpinned it here to show that it remains in the LHN), but yes I think I agree that if the onboarding tasks are created in the Concierge chat, then it just adds extra noise in the LHN to also show the system DM, so it would make sense to hide it.

I agree. I dont think the system chat should be pinned really though so it would only show in the Most Recent as in focus mode it would not be unread

Reply:
@mountiny @francoisl ok, so I added a logic to display Persona only for users onboarded this way. I agree with @francoisl - filtering notifications@expensify.com should be done in a follow-up, as it's something that is occurring on staging 
<img width="1307" alt="Screenshot 2024-06-11 at 09 37 34" src="https://github.com/Expensify/App/assets/28020445/0384b17a-4a22-4270-847e-537840da5d39">
<img width="930" alt="Screenshot 2024-06-11 at 09 37 13" src="https://github.com/Expensify/App/assets/28020445/2678b045-9e08-4313-a806-bb4617b01f40">